src/Jenssegers/Mongodb/Eloquent/EmbedsRelations.php

```diff
@@ -77,4 +78,33 @@ protected function embedsOne($related, $localKey = null, $foreignKey = null, $re
 
         return new EmbedsOne($query, $this, $instance, $localKey, $foreignKey, $relation);
     }
+
+    /**
+     * Define an inverse embedded one-to-one relationship.
+     *
+     * @param  string $related
+     * @param  string $localKey
+     * @param  string $foreignKey
+     * @param  string $relation
+     * @return \Jenssegers\Mongodb\Relations\EmbeddedBy
+     */
+    protected function embeddedBy($related, $foreignKey = null, $ownerKey = null, $relation = null)
+    {
+        // Use debug backtrace to extract the calling method's name and use that as
+        // the relationship name
+        list(, $caller) = debug_backtrace(false);
```

Comment:
i dont understand this, but this looks cool

Reply:
🤷‍♂️